### DISCUSSION TITLE
Will there be a property for getting a user's "about me" section in v13?

### DISCUSSION THREAD

**Discussion by minotaa** [discordjs/discord.js#6105] [Q&A] [answered]
Not to be confused with your custom status, the "bio" or "about me" section of a user's profile.

**Comment by almostSouji** [comment, marked as answer]
You'll have to ask Discord, but currently it looks like they suspect bio data to be identifying, private information which they don't want to be scraped by bots as per https://github.com/discord/discord-api-docs/pull/2951, https://github.com/discord/discord-api-docs/issues/3095

The banner might become accessible soon as per https://github.com/discord/discord-api-docs/issues/3095#issuecomment-868817186 . If this gets documented on the API docs we will include the feature into discord.js at the next possible point (for an additional feature like this a semver minor, so `13.x` or `13.0`, if we haven't released it at that point).

**Reply by minotaa** [reply]
Alright, thanks for the answer
